### ISSUE TITLE
Slow UI redraw and white fringes on HiDPI Linux

### ISSUE THREAD

**Issue by nknganda** [evolus/pencil#190]
* Operating system: Ubuntu 16.04 64-Bit
* Pencil version: 3.0.0-rc1
* Resolution: 3840 x 2160
* Linux 4.4.0-45-generic 4.4.0-45-generic GNU/Linux

When running Pencil on my Dell XPS 15 9550 4k monitor, the UI redraws are very slow and there are white fringes all over.

Every time I move a stencil object to the document canvas, the UI redraws, making it hard to use and when I try an save or exit the application the UI redraws on the save or close popup, so I can't exit the app, have to kill app in terminal.

Might be related to [electron issue](https://github.com/atom/atom/issues/10518)

**Comment by jevonearth** [comment]
Same issue here.

I'm running 
* Archlinux 4.11.9-1-ARCH 
* X windows
* 4K resolution

Pencil doesn't seem to have `--ignore-gpu-blacklist --disable-gpu-sandbox` flags that electron has. I suspect that those settings may help if they were exposed in Pencils CLI flags, or settings.

**Comment by wzin** [comment]
Same here. Dell XPS 9360, resolution: 3200x1800. It makes the app virtually unusable.

**Comment by actionless** [comment]
is there any movement on this?

btw, funny finding -- if i start it with `--enable-dev`, slow redraw issue still persists, however at least white fringes are not appearing anymore

**Comment by toshovski** [comment]
I can also reproduce the same issue on Dell XPS 15 9550 and ubuntu 16.04.

**Comment by Lucretia** [comment]
Same. 3.0.4 on Gentoo.

```
Linux rogue 5.2.0-rc5 #1 SMP PREEMPT Thu Jun 27 15:56:19 BST 2019 x86_64 AMD FX(tm)-8350 Eight-Core Processor AuthenticAMD GNU/Linux
```

**Comment by kokmok** [comment]
solved by cloning the repo and comment lines that disable hardware.
https://github.com/evolus/pencil/issues/55#issuecomment-517240209

**Comment by kokmok** [comment]
I would liek to create a setting for this harware acceleration issue, but can't find where the settings are declared. Any help

**Comment by dgthanhan** [comment]
This was implemented in this commit b89034812edfdb8b0fed1247e6709493b358670a
and is also available in the latest nightly builds at http://pencil.evolus.vn/Nightly.html
